### DISCUSSION TITLE
tailpress_content_start

### DISCUSSION THREAD

**Discussion by merijnponzo** [tailpress/tailpress#54] [Q&A]
Hi Jeffrey,

What does tailpress_content_start do ?

**Comment by jeffreyvr** [comment]
It's just a hook you can use. You can remove it too, it has no impact.

https://developer.wordpress.org/plugins/hooks/

**Reply by kcwoodfield** [reply]
but what does it do?

**Reply by jeffreyvr** [reply]
It does nothing by default, but it allows you to hook some function onto it. Feel free to remove it if you don't want to use it.

Example (you would place this in your functions.php for example):

```php
function your_custom_hook_function() {
  echo 'I always want to show this text before my content section starts';
}
add_action('tailpress_content_start', 'your_custom_hook_function');
```

**Reply by kcwoodfield** [reply]
got it. thank you!
